src/locale/pages/organizeSurvey/en.yml

```diff
@@ -0,0 +1,17 @@
+overview:
```

Comment:
This file (and the entire `src/locale` directory should be taken out. It must be a remnant from before merging `main`?

Reply:
Ah, probably a consequence of me changing some things there before merging with the new localization system.